windows/hid.c

```diff
@@ -306,14 +309,116 @@ int HID_API_EXPORT hid_exit(void)
 	return 0;
 }
 
+static struct hid_device_info* hid_get_device_info(const char* path)
```

Comment:
please keep _consistency_ across the code base, spaces _before_ the asterisk:
```suggestion
static struct hid_device_info *hid_get_device_info(const char *path)
```
this applias to everything below too